mlflow/gateway/providers/clarifai.py

```diff
@@ -0,0 +1,282 @@
+        #                     "user_id": "mistralai",
+        #                     "metadata": {}
+        #                 },
+        #                 "user_id": "mistralai",
+        #                 "model_type_id": "text-to-text",
+        #                 "visibility": {
+        #                     "gettable": 50
+        #                 },
+        #                 "toolkits": [],
+        #                 "use_cases": [],
+        #                 "languages": [],
+        #                 "languages_full": [],
+        #                 "check_consents": [],
+        #                 "workflow_recommended": false
+        #             },
+        #             "input": {
+        #                 "id": "4341d6394ba245f1ab9d8a0d7ba75939",
+        #                 "data": {
+        #                     "text": {
+        #                         "raw": "<s><INST>I love your product very much</INST>",
+        #                         "url": "https://samples.clarifai.com/placeholder.gif"
+        #                     }
+        #                 }
+        #             },
+        #             "data": {
+        #                 "text": {
+        #                     "raw": "That's great to hear! We're delighted that you're enjoying it.
+        #                             If there's anything else you need or any feedback
+        #                             you'd like to share, please let us know.",
+        #                     "text_info": {
+        #                         "encoding": "UnknownTextEnc"
+        #                     }
+        #                 }
+        #             }
+        #         }
+        #     ]
+        # }
+        # ```
+        return completions.ResponsePayload(
+            created=int(time.time()),
+            object="text_completion",
+            model=config.model.name,
+            choices=[
+                completions.Choice(
+                    index=0,
+                    text=resp["outputs"][-1]["data"]["text"]["raw"],
+                    finish_reason=None,
+                )
+            ],
+            usage=completions.CompletionsUsage(
+                prompt_tokens=None,
+                completion_tokens=None,
+                total_tokens=None,
+            ),
+        )
+
+    @classmethod
+    def completions_to_model(cls, payload, config):
+        if payload["n"] != 1:
+            raise HTTPException(
+                status_code=422,
+                detail="Only one generation is supported. Please set candidate_count to 1.",
+            )
+
+        params = {}
+        if payload.get("temperature"):
+            params["temperature"] = payload["temperature"]
+        if payload.get("max_tokens"):
+            params["max_tokens"] = payload["max_tokens"]
+        return {
+            "inputs": [{"data": {"text": {"raw": payload["prompt"]}}}],
+            "model": {"output_info": {"params": params}},
+        }
```

Comment:
```suggestion
        params = {}
        if temperature := payload.get("temperature"):
            params["temperature"] = temperature
        if max_tokens := payload.get("max_tokens"):
            params["max_tokens"] = max_tokens
        return {
            "inputs": [{"data": {"text": {"raw": payload["prompt"]}}}],
            "model": {"output_info": {"params": params}},
        }
```
Consistency with our coding style